bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java

```diff
@@ -195,30 +196,30 @@ public boolean equals(Object obj) {
   private SerializableScan serializableScan;
 
   /**
-   * Creates a {@link CloudBigtableScanConfiguration} using the specified information.
+   * Creates a {@link CloudBigtableScanConfiguration} using the specified configuration.
```

Comment:
"Creates a {@link CloudBigtableScanConfiguration} that will perform the specified scan in the table."
?